app/services/animation.js

```diff
@@ -0,0 +1,34 @@
+import Service from '@ember/service';
+import { computed } from '@ember/object';
+import fade from 'ember-animated/transitions/fade';
+import config from 'travis/config/environment';
+
+const { environment } = config;
+const isTest = environment === 'test';
```

Comment:
Another way of checking for testing is `if (Ember.testing) ...`, though, I'm not sure if it's recommended, just shorter 😄Current implementation works perfectly though, comment is mostly for information ;)

Reply:
Ahh nice! I think I like that better, so I switched to it. Thanks for letting me know!